cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -437,7 +437,7 @@ def get_full_sha_from_short(short_sha):
 
 
 def get_author_info_from_short_sha(short_sha):
-    cmd = f"git log -1 --format='%aN <%ae>' {short_sha}"
+    cmd = f'git log -1 --format="%aN <%ae>" {short_sha}'
```

Comment:
I have had cherry_picker -- continue work for me, but I am using the original version, which does not have this code ;-).  I know that double quotes are required for Windows commands.  I have the impression that single quotes are required for Linux.  This is the wrong fix.   The subprocess doc says

"args is required for all calls and should be a string, or a sequence of program arguments. Providing a sequence of arguments is generally preferred, as it allows the module to take care of any required escaping and quoting of arguments (e.g. to permit spaces in file names). If passing a single string, either shell must be True (see below) or else the string must simply name the program to be executed without specifying any arguments."

When neither of the last two two conditions is true, the idea is to let subprocess handle the OS-specific escaping and quoting.  (At least on Windows, a string properly formatted for Windows does work, and I know people use them.)  It seems that args should be a list.  I am not sure what constitutes a 'program argument', in in particular, how one is 'supposed to' handle '--argname=value', but maybe this works on both Linux and *nix.

`cmd = ['git', 'log', '-1', '--format=', '%aN <%ae>', f'{short_sha}']

Reply:
I agree with Terry, although I haven't tested this particular case on Windows.

If using a list for `cmd`, it should be:
`cmd = ['git', 'log', '-1', '--format=%aN <%ae>', f'{short_sha}']`

Since the shell wouldn't split `--argname=value`, it should remain a single item in the list.

Reply:
I followed @ericvsmith suggestion and I can confirm it works on Windows.